tests/test-json-responses.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Test case for Basic Router responses.
+ *
+ * @group ajax
+ */
+class WPGraphQL_Test_Router extends WP_Ajax_UnitTestCase {
+
+	public function setUp() {
+		parent::setUp();
+	}
+
+	public function tearDown() {
+		parent::tearDown();
+	}
+
+	public function testResolveHttpRequestWithEmptyQuery() {
+		/**
+		 * Filter the request data
+		 */
+		add_filter( 'graphql_request_data', function( $data ) {
+			$data['query'] = null;
+			$data['variables'] = null;
+			$data['operationName'] = null;
+			return $data;
+		} );
+
+		/**
+		 * Set the query var to "graphql" so we can mock like we're visiting the endpoint via
+		 */
+		set_query_var( 'graphql', true );
+		$GLOBALS['wp']->query_vars['graphql'] = true;
+
+		/**
+		 * Instantiate the router
+		 */
+		$router = new \WPGraphQL\Router();
```

Comment:
Looks like this is never used, as the use of it is commented out?

Reply:
This gets used by the hook and I think it needs to be instantiated earlier.